packages/experiments/src/components/PersonaCoin/PersonaCoin.state.ts

```diff
@@ -1,28 +1,30 @@
+import { useCallback, useState } from 'react';
 import { ImageLoadState } from 'office-ui-fabric-react';
-import { IPersonaCoinProps, IPersonaCoinViewProps } from './PersonaCoin.types';
-import { BaseState } from '../../utilities/BaseState';
+import { IPersonaCoinViewProps, IPersonaCoinComponent } from './PersonaCoin.types';
 
-export type IPersonaCoinState = Pick<IPersonaCoinViewProps, 'isPictureLoaded' | 'onPhotoLoadingStateChange'>;
+export const usePersonaCoinState: IPersonaCoinComponent['state'] = props => {
+  // TODO: isPictureLoaded was controlled, does it need to be? it's not exposed through component props...
+  //       For now use useState.
+  const [isPictureLoaded, setIsPictureLoaded] = useState(false);
```

Comment:
@Markionium Did you intend `isPictureLoaded` to be a controlled prop? Since it's not part of `IPersonaCoinProps` it couldn't actually be controlled without users generating a type error.

Reply:
Yes that seems correct afaik.

Persona uses the Image component and we want to show the initials before showing you the picture. It might not make much sense to allow the user of the component determine some custom logic for how that works.

If we'll need to do that at some point in the future thats probably fine, but for now i think it's ok to keep this controlled.

The user can listen to updates in the loading state by using the `onPhotoLoadingStateChange` callback.